src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -52,6 +52,13 @@
                     @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTotalSpansHeader)] <strong>@trace.Spans.Count</strong>
                 </div>
 
+                <FluentButton Appearance="Appearance.Lightweight" OnClick="@OnCollapseAllAsync" title="@Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailCollapseAll)]">
+                    <FluentIcon Icon="Icons.Regular.Size16.ArrowCollapseAll" />
+                </FluentButton>
+                <FluentButton Appearance="Appearance.Lightweight" OnClick="@OnExpandAllAsync" title="@Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailExpandAll)]">
```

Comment:
src/Aspire.Dashboard/Components/Pages/TraceDetail.razor(58,146): error CS0117: (NETCORE_ENGINEERING_TELEMETRY=Build) 'TraceDetail' does not contain a definition for 'TraceDetailExpandAll'